### PR TITLE
💾 Local Storage Session Saving

### PR DESCRIPTION
**Pull request by benjdlambert**
### Problem
- When users navigate away from onboarding their progress is lost

### Solution
- Populate the session of onboarding with the data that is already in Local Storage

creator

### Notes
https://trello.com/c/pDY3oeY8
@revett @stevenjack

### CODE REVIEW

**Comment by benjdlambert** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;pDY3oeY8&#x2F;345-cr-sa-use-local-storage-html5-to-save-onboarding-state"> 👴👴 [CR] [SA] Use local-storage [HTML5] to save onboarding state</a>

File: source/jsx/creator/form/input-field.jsx [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+    errors: React.PropTypes.arrayOf(React.PropTypes.string),
+    helper: React.PropTypes.oneOfType([
+      React.PropTypes.node,
+      React.PropTypes.arrayOf(React.PropTypes.node),
+    ]),
+  }
+  static defaultProps = {
+    type: 'text',
+    errors: [],
+    onBluer: () => undefined,
```
Comment (revett) [inline source/jsx/creator/form/input-field.jsx]: Typo 💥

File: source/jsx/creator/store.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const STORE_NAME = 'onboarding.formstate';
```
Comment (revett) [inline source/jsx/creator/store.jsx]: 🏄 

Prefix with `vidsy.`?

**Comment by revett** [comment]
🍰

File: source/jsx/creator/store.jsx (hunk 2 above)
Comment (benjdlambert) [inline source/jsx/creator/store.jsx]: `local-storage` is scoped on a per domain basis anyway for security.

So I thought `onboarding` was a good namespace? Up to you tho, happy either way.

**Pull request merged**